packages/plugin-eslint-rules/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@grafana/plugin-eslint-rules",
+  "version": "0.0.4",
+  "description": "An ESlint plugin with Grafana plugin specific rules",
+  "main": "./dist/index.js",
+  "files": [
+    "dist/"
+  ],
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
```

Comment:
I forgot to mention that the other packages in this repo use provenance. We can add the `provenance` property to `publishConfig` as mentioned in the [docs here](https://docs.npmjs.com/generating-provenance-statements#using-third-party-package-publishing-tools). (Or take a peek in one of the other packages package.json.

Reply:
Fixed 👍